### PR TITLE
data: ShubhamChikane99

### PR DESCRIPTION
**Pull request by ShubhamChikane99**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [ ] My change requires changes to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers
You'll are doing great job.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4793"></a>

### CODE REVIEW

**Review by accodes21 — APPROVED** [review]

**Review by priyankarpal — APPROVED** [review]
LGTM 🎉

File: data/ShubhamChikane99.json [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+      "group": "Primary Socials",
+      "name": "Twitter: Follow me",
+      "url": "https://twitter.com/shubhamchikane_",
+      "icon": "FaTwitter",
+      "color": "#00ACEE"
+    },
+   {
+      "group": "Primary Socials",
+      "name": "LinkedIn: Connect with me ",
+      "url": "https://linkedin.com/in/shubham chikane",
```
Comment (priyankarpal) [inline data/ShubhamChikane99.json:29]: hey @ShubhamChikane99, your LinkedIn profile doesn't exist. Please add a valid URL. 
✌️

**Review by priyankarpal — CHANGES_REQUESTED** [review]

File: data/ShubhamChikane99.json (hunk 1 above)
Comment (Pradumnasaraf) [reply data/ShubhamChikane99.json:29]: It's not a blocker, the fix can come in another PR. merging it now

Thank you

**Review by Pradumnasaraf — CHANGES_REQUESTED** [review]
Hey, there is merge conflict please fix that

**Pull request closed without merging**